<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultSetting</class>
 <widget class="QWidget" name="ResultSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResultSetting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:yellow;　　　　　　　　　　　　 
　　　　　　　　

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_25">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font-size:15px; </string>
     </property>
     <property name="title">
      <string>线头参数设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-size:15px; </string>
          </property>
          <property name="text">
           <string>对齐度报警值:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBox_head_alignment">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">font-size:15px; </string>
          </property>
          <property name="text">
           <string>对齐度纠偏值:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBox_head_alignment_2">
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>0.300000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>超限报警:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>左标准值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_head_left_warning">
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>50.000000000000000</double>
            </property>
            <property name="value">
             <double>27.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>中标准值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_head_middle_warning">
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>50.000000000000000</double>
            </property>
            <property name="value">
             <double>199.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>右标准值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_head_right_warning">
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
            <property name="value">
             <double>27.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLineEdit" name="lineEdit_head_test"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_head_show"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px; </string>
     </property>
     <property name="title">
      <string>线尾参数设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>超限报警:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>中标准值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_end_middle_warning">
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>199.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font-size:15px; </string>
          </property>
          <property name="text">
           <string>对齐方式设置:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="radioButton_left">
            <property name="text">
             <string>左对齐</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_middle">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>中对齐</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_right">
            <property name="text">
             <string>右对齐</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLineEdit" name="lineEdit_end_text"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_end_show"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>对齐度报警值:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font-size:15px; </string>
            </property>
            <property name="text">
             <string>左对齐限值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_end_left_alignment">
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">font-size:15px; </string>
            </property>
            <property name="text">
             <string>居中对齐限值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_end_middle_alignment">
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">font-size:15px; </string>
            </property>
            <property name="text">
             <string>右对齐限值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_end_right_alignment">
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="0,0,0">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>对齐度纠偏值:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="styleSheet">
             <string notr="true">font-size:15px; </string>
            </property>
            <property name="text">
             <string>左对齐限值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_end_left_alignment_3">
            <property name="value">
             <double>0.300000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="styleSheet">
             <string notr="true">font-size:15px; </string>
            </property>
            <property name="text">
             <string>右对齐限值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinBox_end_right_alignment_3">
            <property name="value">
             <double>0.300000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
